mithril-aggregator/src/certifier_service.rs

```diff
@@ -170,7 +170,7 @@ impl MithrilCertifierService {
 impl CertifierService for MithrilCertifierService {
     async fn inform_epoch(&self, epoch: Epoch) -> StdResult<()> {
         debug!("CertifierService::inform_epoch(epoch: {epoch:?})");
-        let nb = self.open_message_repository.clean_epoch(epoch).await?;
+        let nb = self.open_message_repository.clean_epoch(epoch - 1).await?;
```

Comment:
maybe use a strict lower than instead of passing `epoch-1` ?

Reply:
Done :heavy_check_mark: